### PR TITLE
Add support for Bitcoin CSV

### PR DESCRIPTION
**Pull request by LeoComandini**

### CODE REVIEW

**Comment by jgriffiths** [comment]
utack aca080e6deca6c444620c6bba176f6ff0f4aaf1c

File: README.md [hunk 1]
```diff
@@ -141,6 +177,41 @@ If any recoverable coins were found the tool will display a summary of
 them on the console and write the details to the output csv file ready for
 broadcasting using the same steps as detailed above for 2of2 subaccounts.
 
+## Liquid Recovery
+
+In the case of Liquid subaccounts the outputs are locked by CSV
+(OP_CHECK_SEQUENCE_VERIFY) scripts. Which means that coins can be spent either
+with by signing with your default key and the Green/GreenAddress key under
```
Comment (jkauffman1) [inline README.md:184]:
```suggestion
by signing with your default key and the Green/GreenAddress key under
```

**Comment by jkauffman1** [comment]
utack ff52848eb756e8f471d76f16e29d058978081e1f

File: README.md (hunk 1 above)
Comment (Shuhaib07) [reply README.md]: 1139186

**Pull request merged**